extra/1-currency-conversion.js

```diff
@@ -15,7 +17,12 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  let newPrice = (99 * price) / 100;
```

Comment:
`newPrice` needs to be 1% less than `price`. To do that you could multiply it by a number less than 1, to reduce its value. Eg multiplying `price` by 0.75 will give you 75% of its value.